client/components/payment-process/index.css

```diff
@@ -0,0 +1,10 @@
+.payment-buffer .main-column {
+  padding: 10px;
+  border-radius: 5px;
+}
+
+iframe {
+  width: 100%;
+  height: 500px;
+  border-radius: 5px;
```

Comment:
I would add `border: 0 none;` to avoid making it super obvious that it's an `iframe`.